conformance/conformance_test.cc

```diff
@@ -771,13 +775,60 @@ bool ConformanceTestSuite::RunSuite(ConformanceTestRunner* runner,
   ExpectParseFailureForJson(
       "TrailingCommaInAnObject",
       R"({"fieldname1":1,})");
+  ExpectParseFailureForJson(
+      "TrailingCommaInAnObjectWithSpace",
+      R"({"fieldname1":1 ,})");
```

Comment:
Are there any implementations passing "TrailingCommaInAnObject" but failing this?